integrated_channels/integrated_channel/management/commands/transmit_learner_data.py

```diff
@@ -0,0 +1,84 @@
+"""
+Transmits consenting enterprise learner data to the Enterprise Integrated Channel SAP SuccessFactors.
+"""
+
+import logging
+from django.core.management.base import BaseCommand, CommandError
+
+
+from enterprise.models import EnterpriseCustomer
+from integrated_channels.sap_success_factors.models import SAPSuccessFactorsEnterpriseCustomerConfiguration
+
+
+LOGGER = logging.getLogger(__name__)
+
+class Command(BaseCommand):
+    """
+    Management command which transmits learner course completion data to the IntegratedChannel(s) configured for the
+    given EnterpriseCustomer.
+
+    Collect the enterprise learner data for enrollments with data sharing consent, and transmit each to the
+    EnterpriseCustomer's configured IntegratedChannel(s).
+    """
+
+    help = """
+    Transmit enterprise learner course completion data for the given enterprise customer.
+    """
+    integrated_channel_choices = {
+        'sap': SAPSuccessFactorsEnterpriseCustomerConfiguration,
+    }
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '--enterprise_customer',
+            dest='enterprise_customer',
+            metavar='ENTERPRISE_CUSTOMER_UUID',
+            help='Transmit data for this EnterpriseCustomer to its configured integration channel(s).',
+        )
+        parser.add_argument(
+            '--channel',
+            dest='channel',
+            default=None,
+            metavar='INTEGRATED_CHANNEL',
+            help='Transmit data to this INTEGRATED_CHANNEL.  '
+            'Omit option to transmit to all configured, active integration channels.',
+            choices=self.integrated_channel_choices.keys(),
+        )
+
+    def handle(self, *args, **options):
+        """
+        Collect the learner data for the given EnterpriseCustomer, and transmit it to its active integration channels.
+        """
+        # Ensure there's a valid enterprise customer
+        enterprise_customer_uuid = options['enterprise_customer']
+        if not enterprise_customer_uuid:
+            raise CommandError('Enterprise customer uuid required.')
```

Comment:
@pomegranited, I was actually talking to @brittneyexline, and it looks like for Jenkins, we need to run one single command to do everything - we can't run multiple instances with EC UUIDs as arguments. I'll get a more thorough review in later today, but we'd like to:

1. Accept EC UUIDs as an _optional_ parameter, with behavior as seen here
2. When an EC UUID is _not_ provided, retrieve a list of EnterpriseCustomers, and perform the task for each EC UUID in separate Celery tasks

Sorry for the late-breaking change; we've been simultaneously talking with Ops about how the EdX Jenkins tasks run.